doc/api/tls.md

```diff
@@ -520,6 +530,41 @@ determine if the server certificate was signed by one of the specified CAs. If
 `tlsSocket.alpnProtocol` property can be checked to determine the negotiated
 protocol.
 
+The `'secureConect'` event is only emitted on `tls.TLSSocket` connections
+created using the `tls.connect()` method.
+
+### Event: 'secure'
+<!-- YAML
+added: REPLACEME
```

Comment:
@sam-github @bnoordhuis ... off the top of your heads... do you know what version this was added?

Reply:
v0.11.3, the introduction of `TLSSocket`. `SecurePair` before that emitted it too though, so in a way it goes all the way back to v0.3.2.